src/Common/Admin/Help_Hub/Hub.php

```diff
@@ -465,6 +467,10 @@ public function enqueue_help_page_iframe_assets(): void {
 				],
 			]
 		);
+
+		tribe_asset_enqueue( 'help-hub-iframe-style' );
+		tribe_asset_enqueue( 'help-hub-iframe-js' );
```

Comment:
```suggestion
		tribe_asset_enqueue( 'tec-help-hub-iframe-style' );
		tribe_asset_enqueue( 'tec-help-hub-iframe-js' );
```
We should add prefix here.